.github/workflows/unitpacker-go.yml

```
@@ -0,0 +1,255 @@
+    strategy:
+      matrix: ${{ fromJSON(needs.config.outputs.config) }}
+    runs-on: ${{ matrix.os }}
+    needs:
+      - config
+      - build
+      - sign
+    steps:
+
+      - uses: actions/checkout@v3
+        if: runner.os == 'macOS'
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: ${{ github.run_id }}.${{ matrix.os }}.signed
+
+      - name: Create Linux ZIP
+        if: runner.os == 'Linux'
+        run: |
+          mv ${{ matrix.tools.name }}.asc ${{ matrix.tools.name }}_${{ matrix.tools.ref }}_linux.asc
+          zip ${{ matrix.tools.name }}_${{ matrix.tools.ref }}_linux.zip ${{ matrix.tools.name }} ${{ matrix.tools.name }}_${{ matrix.tools.ref }}_linux.asc
+
+      - uses: apple-actions/import-codesign-certs@v1
+        if: runner.os == 'macOS'
+        with:
+          p12-file-base64: ${{ secrets.MAC_INST_CERTIFICATE }}
+          p12-password: ${{ secrets.MAC_INST_CERTIFICATE_PASSPHRASE }}
+
+      - name: Create folder structure, package and sign
+        if: runner.os == 'macOS'
+        run: |
+          wget -q -O stemplate https://github.com/freshautomations/stemplate/releases/download/v0.6.1/stemplate_darwin_amd64
+          chmod 755 stemplate multisig
+          mkdir -p package/usr/local/bin package/usr/local/lib/${{ matrix.tools.name }}
+          export NAME=${{ matrix.tools.name }}
+          export VERSION=${{ matrix.tools.ref }}
+          ./stemplate Info.plist.template --string NAME,VERSION --output package/usr/local/lib/${{ matrix.tools.name }}/Info.plist
+          mv ${{ matrix.tools.name }} package/usr/local/bin
+          pkgbuild --root package ${{ matrix.tools.name }}_${{ matrix.tools.ref }}_mac.pkg --timestamp --sign "${{ secrets.MAC_INST_CERTIFICATE_CN }}"
+
+      - name: Notarize package
+        if: runner.os == 'macOS'
+        run: |
+          OUT="$(xcrun notarytool submit ${{ matrix.tools.name }}_${{ matrix.tools.ref }}_mac.pkg --wait -f json --apple-id "${{ secrets.MAC_STORE_APPLE_ID }}" --password "${{ secrets.MAC_STORE_APP_SPECIFIC_PASSWORD }}" --team-id "${{ secrets.MAC_STORE_TEAM_ID }}")"
+          echo "$OUT"
+          STATUS="$(echo "$OUT" | jq -r .status)"
+          ID="$(echo "$OUT" | jq -r .id)"
+          if [ "$STATUS" != "Accepted" ]; then exit 1 ; fi
+          LOG="$(xcrun notarytool log "$ID" --apple-id "${{ secrets.MAC_STORE_APPLE_ID }}" --password "${{ secrets.MAC_STORE_APP_SPECIFIC_PASSWORD }}" --team-id "${{ secrets.MAC_STORE_TEAM_ID }}")"
+          echo "$LOG"
+          ISSUES="$(echo "$LOG" | jq -r .issues)"
+          if [ "$ISSUES" != "null" ]; then exit 1 ; fi
+
+      - name: Staple package
+        if: runner.os == 'macOS'
+        run: |
+          xcrun stapler staple ${{ matrix.tools.name }}_${{ matrix.tools.ref }}_mac.pkg
+          xcrun stapler validate ${{ matrix.tools.name }}_${{ matrix.tools.ref }}_mac.pkg
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: ${{ github.run_id }}.macos-12.signed
```

Something's fishy here: `mac-12` will break when GitHub Actions upgrades to macos-13, macos-14, etc. Also, we have to upload any Linux-generated files too.

Is this what you had in mind?
```suggestion
          name: ${{ github.run_id }}.${{ matrix.os }}.signed
```

---

This was missed. It is updated now.